scripts/ci-workflows.test.mjs

```
@@ -831,3 +838,91 @@ test('both native platforms upload Maestro artifacts unconditionally', () => {
+			.runFlow.commands;
+
+		assert.deepEqual(
+			androidLaunch[0],
+			{ tapOn: { text: 'Wait', optional: true } },
+			`${filename} must clear an ANR dialog that predates hide_error_dialogs`
+		);
+	}
+});
+
+test('the Android step retries a transient offline ADB transport once', () => {
+	const step = findStep(readWorkflow('e2e-native.yml'), 'android', '📱 Run Maestro suite on emulator');
```

ESLint

[error] Replace `readWorkflow('e2e-native.yml'),·'android',·'📱·Run·Maestro·suite·on·emulator'` with `⏎↹↹readWorkflow('e2e-native.yml'),⏎↹↹'android',⏎↹↹'📱·Run·Maestro·suite·on·emulator'⏎↹`

(prettier/prettier)